docpage/docs/Users.md

```diff
@@ -40,11 +40,11 @@ python dcnios-cli.py changeSchedule --host={nifi-endpoint} \
 
 ## File workflow configuration structure (Yaml structure)
 
-Here, we will explain the workflow definition, the structure of the configuration file, and the information the user has to know about each third-party connection. DCNiOS deploys and configures all the definitions in Apache NiFi.
+Here, we explain the workflow definition, the structure of the configuration file, and the information the user has to know about each third-party connection. DCNiOS deploys and configures all the definitions in Apache NiFi.
 
 ### Apache NiFi credentials:
 
-In this 'nifi' section, the Apache NiFi credentials will be defined. Inside this section will be defined the Sources that will be deployed and the conection between them.
+In this `nifi` section, set the Apache NiFi credentials. Inside this section, define the workflow.
```

Comment:
In this `nifi` section, the Apache NiFi credentials and the workflow including sources, destinations, and connections are defined.

*add* something like this in the example yaml:
```
<Source>:
... 
<Destination>:
...
connection:
...
```